syllabus/7-Polkadot/13-polkadot-ecosystem-economy-slides.md

```diff
@@ -0,0 +1,652 @@
+| 11  |     Treasurer      |     1000 DOT     |    2 Hours     |     28 Days     |    3 Hours     |        1 Day         |
+| 12  |    Lease Admin     |     5000 DOT     |    2 Hours     |     28 Days     |    3 Hours     |      10 Minutes      |
+
+---
+
+## Approval and Support Curves
+
+Each track has their own set of curves which determine if the proposal has passed or failed.
+
+All votes will eventually resolve one way or another.
+
+<image src="../../assets/img/7-Polkadot/eco/pjs-curves.png" style="width: 1000px;">
+
+You can find these curves on the Polkadot JS Developer Console.
+
+---
+
+## Example: Root
+
+The origin with the highest level of privileges. Requires extremely high levels of approval and support for early passing. The prepare and enactment periods are also large.
+
+<image src="../../assets/img/7-Polkadot/eco/root-curve.png" style="width: 800px;">
+
+For instance, a referendum proposed in this track needs to amass 48.2% support (total network issuance) by the end of the first day with over 93.5% approval to be considered to be part of the confirm period.
+
+---
+
+## Governance Token Mechanics
+
+- DOT tokens are locked when you vote on a proposal.
+- You can reuse your locked tokens across multiple proposals.
+  - Voting for one proposal does not affect your ability to vote on another proposal.
+- You can also reuse staked tokens (which are also just locked).
+- You can update your vote while the proposal is ongoing.
+- If you used conviction voting, your tokens might be locked for long periods of time passed the end of a proposal.
+
+---
+
+## Treasury
+
+Polkadot has an on-chain treasury which is used to support permissionless and decentralized development of the network.
+
+Treasury gets its funds through inefficiencies in the inflation curve, slashing, and by taking 80% transaction fees.
+
+Treasury will automatically burn a 1% of its funds every spending period (24 days), placing pressure on spending the tokens.
+
+---
+
+## Treasury Outlets
+
+- Proposals: Instant payments to individuals approved by governance.
+- Bounties: Multi-stage payments to individuals managed by governance and an appointed bounty curator.
+- Tips: Small payments to individuals that can be made more easily through specific governance tracks.
+
+The Polkadot Treasury currently has over 33,000,000 DOT.
+
+---
+
+# Ecosystem
+
+---
+
+## Alternative Polkadot Clients
+
+The main Polkadot Client is built using Rust in Substrate.
+
+However, other clients for Polkadot are under development:
+
+- Kagome (C++17): https://github.com/soramitsu/kagome
+- Gossamer (Go): https://github.com/ChainSafe/gossamer
+
+Over time, this can help provide additional resilience to the network from software bugs.
+
+---
+
+## Types of Parachains
+
+- Polkadot System Chains
+- Market Bought Parachains
+
+<br />
+
+Perhaps this list will grow once more agile core allocation systems are implemented.
```

Comment:
Still accurate I guess but coretime allows to have parachains (tasks to be more accurate) produce sporadically or flexibly instead of just 1 block per 12 sec